Monika After Story/game/script-story-events.rpy

```diff
@@ -2077,37 +2077,55 @@ label mas_notification_windowreact:
+            m 3rksdla "And maybe if you install that, you could install the xdotool command too..."
+            m 3eua "That would allow me to see your active window!"
+            $ dlg_line = "Of course, you don't have to install the second one, but"
+
         show monika 5eka at t11 zorder MAS_MONIKA_Z with dissolve
-        m 5eka "...And I'd really appreciate it, [player]."
-        return
+        m 5eka "...[dlg_line] I'd really appreciate it, [player]."
 
-    m 3eub "Would you like to see how they work?{nw}"
-    $ _history_list.pop()
-    menu:
-        m "Would you like to see how they work?{fast}"
-
-        "Sure!":
-            m 1hua "Okay, [player]!"
-            m 2dsa "Just give me a second to make a notification.{w=0.5}.{w=0.5}.{nw}"
-            $ display_notif(m_name, ["I love you, [player]!"], skip_checks=True)
-            m 1hub "There it is!"
-
-        "No thanks.":
-            m 2eka "Alright, [player]."
-
-    m 3eua "If you want me to notify you, just head over to the 'Alerts' tab in the settings menu and turn them on, along with what you'd like to be notified for."
-
-    if renpy.windows:
-        m 3rksdla "Also, since you're using Windows...I now know how to check what your active window is..."
-        m 3eub "So if I have something to talk about while I'm in the background, I can let you know!"
-        m 3hksdlb "And don't worry, I know you might not want me constantly watching you, and I respect your privacy."
-        m 3eua "So I'll only look at what you're doing if you're okay with it."
-        m 2eua "If you enable 'Window Reacts' in the settings menu, that'll tell me you're fine with me looking around."
-
-        if mas_isMoniNormal(higher=True):
-            m 1tuu "It's not like you have anything to hide from your girlfriend..."
-            show monika 5ttu at t11 zorder MAS_MONIKA_Z with dissolve
-            m 5ttu "...right?"
+    else:
+        m 3eub "Would you like to see how they work?{nw}"
+        $ _history_list.pop()
+        menu:
+            m "Would you like to see how they work?{fast}"
+
+            "Sure!":
+                m 1hua "Okay, [player]!"
+                m 2dsa "Just give me a second to make a notification.{w=0.5}.{w=0.5}.{nw}"
+                $ display_notif(m_name, ["I love you, [player]!"], skip_checks=True)
+                m 1hub "There it is!"
+
+            "No thanks.":
+                m 2eka "Alright, [player]."
+
+        m 3eua "If you want me to notify you, just head over to the 'Alerts' tab in the settings menu and turn them on, along with what you'd like to be notified for."
+
+        if renpy.windows:
+            m 3rksdla "Also, since you're using Windows...I now know how to check what your active window is..."
+
+
+        elif renpy.linux:
+            if mas_windowreacts.can_do_windowreacts:
+                m 3rksdla "Also, since you have the xdotool command installed...I now know how to check what your active window is..."
```

Comment:
```suggestion
                m 3rksdla "Also, since you have the xdotool command installed...I now know how to check what your active window is."
```